src/service/performance-impl.js

```diff
@@ -353,6 +361,8 @@ export class Performance {
         if (!entry.hadRecentInput) {
           this.aggregateShiftScore_ += entry.value;
         }
+      } else if (entry.entryType === 'largest-contentful-paint') {
```

Comment:
My only concern here is that using the single value startTime might be misleading in some cases. For images that do not pass Timing-Allow-Origin checks, the startTime will be the load time, not the render time. For images that do pass the checks, the startTime will be the render time. Since load time < render time, this is essentially going to penalize cases where the image has the appropriate headers to measure the timing more accurately. If we want to have a single number, that's the best we can do in RUM, but otherwise maybe we could somehow split these cases? That is, cases where renderTime is present vs cases where it's not.

Reply:
Hmm - I think the solution you're implying would be to split the two cases as you suggest into  two metrics - `lcpl` for the presence of load time and `lcpr` for the presence of render time. 

I think that would work for our analysis - what do you think?

Reply:
I think that makes sense!

Reply:
Changed this to two metrics